Packs/AWS-CloudTrail/Integrations/AWS-CloudTrail/AWS-CloudTrail.py

```diff
@@ -373,49 +412,44 @@ def lookup_events(args: dict) -> CommandResults:
 
-    return CommandResults(
-        outputs_prefix="AWS.CloudTrail.Events",
-        outputs_key_field="EventId",
-        outputs=data,
-        readable_output=tableToMarkdown('AWS CloudTrail Trails', data),
-    )
+    ec = {'AWS.CloudTrail.Events(val.EventId == obj.EventId)': data}
+    human_readable = tableToMarkdown('AWS CloudTrail Trails', data)
+    return_outputs(human_readable, ec)
 
 
-def test_function() -> str:
+def test_function():
     client = aws_session()
     response = client.describe_trails()
-    if (response_code := response['ResponseMetadata']['HTTPStatusCode']) == 200:
-        return "ok"
-    raise DemistoException(f"Unexpected status code: {response_code}")
-
-
-def main():
-    command = demisto.command()
-    args = demisto.args()
-    try:
-        if command == 'test-module':
-            return_results(test_function())
-        if command == 'aws-cloudtrail-create-trail':
-            return_results(create_trail(args))
-        if command == 'aws-cloudtrail-delete-trail':
-            return_results(delete_trail(args))
-        if command == 'aws-cloudtrail-describe-trails':
-            return_results(describe_trails(args))
-        if command == 'aws-cloudtrail-update-trail':
-            return_results(update_trail(args))
-        if command == 'aws-cloudtrail-start-logging':
-            return_results(start_logging(args))
-        if command == 'aws-cloudtrail-stop-logging':
-            return_results(stop_logging(args))
-        if command == 'aws-cloudtrail-lookup-events':
-            return_results(lookup_events(args))
-
-    except Exception as e:
-        err = "Error has occurred in the AWS CloudTrail Integration."
-        if isinstance(e, ResponseParserError):
-            err += " Could not connect to the AWS endpoint. Please check that the region is valid."
-        return_error(f"{err}\nError: {e}")
-
-
-if __name__ in ["__builtin__", "builtins", "__main__"]:
-    main()
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results('ok')
+
+
+'''EXECUTION BLOCK'''
+try:
+    if demisto.command() == 'test-module':
+        test_function()
+    if demisto.command() == 'aws-cloudtrail-create-trail':
+        create_trail(demisto.args())
+    if demisto.command() == 'aws-cloudtrail-delete-trail':
+        delete_trail(demisto.args())
+    if demisto.command() == 'aws-cloudtrail-describe-trails':
+        describe_trails(demisto.args())
+    if demisto.command() == 'aws-cloudtrail-update-trail':
+        update_trail(demisto.args())
+    if demisto.command() == 'aws-cloudtrail-start-logging':
+        start_logging(demisto.args())
+    if demisto.command() == 'aws-cloudtrail-stop-logging':
+        stop_logging(demisto.args())
+    if demisto.command() == 'aws-cloudtrail-lookup-events':
+        lookup_events(demisto.args())
+    if demisto.command() == 'aws-cloudtrail-get-trail-status':
+        get_trail_status(demisto.args())
```

Comment:
Why did you remove the main()?
Please revert this change.
Also, the video you send me is not working for some reason.. please attach a new one.
Let's make sure to whole build flow is passing and that the video is attached and I'll move on with the pr.